test/e2e/.chromedriver_version

```diff
@@ -1 +1 @@
-77.0.3865.40
+79.0.3945.36
```

Comment:
@blowery: I think we should also update the `chromedriver` version number in `test/e2e/package.json` (similarly to what was done in https://github.com/Automattic/wp-calypso/pull/36978). I'm indeed getting the following errors [with E2E](https://circleci.com/gh/Automattic/wp-e2e-tests-for-branches/39498) tests for https://github.com/Automattic/wp-calypso/pull/38333:

```
session not created: This version of ChromeDriver only supports Chrome version 77
```

Reply:
Thanks! https://github.com/Automattic/wp-calypso/pull/38508

Reply:
Fixed on master.

Reply:
Thanks Ben!